lib/charms/grafana_cloud_integrator/v0/cloud_config_provider.py

```diff
@@ -58,3 +61,4 @@ def _on_relation_changed(self, event):
                 databag["loki_url"] = self._loki_url
             if self._charm.prom_configured:
                 databag["prometheus_url"] = self._prometheus_url
+            databag["tls-ca"] = self._charm.config.get("tls-ca", "")
```

Comment:
This effectively obviates the send-ca-cert relation. Cool. I think.

Reply:
Done!